.github/workflows/build_and_test.yml

```
@@ -5,11 +5,15 @@ on: [push, pull_request]
 
 jobs:
   build_wheels:
-    name: Build wheels on ${{ matrix.os }}
+    name: Build wheels on ${{matrix.arch}} for ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ubuntu-18.04, windows-latest, macos-latest]
+        arch: [auto]
+        include:
+          - os: ubuntu-18.04
+            arch: aarch64
```

Could this be:

```suggestion
        include:
          - os: ubuntu-18.04
            archs: auto
          - os: ubuntu-18.04
            archs: aarch64
          - os: windows-latest
            archs: auto
          - os: macos-latest
            archs: auto universal2
```

Maybe easier just to spell it out... Also, I renamed `arch` -> `archs` to match cibuildwheel's option name.